### PR TITLE
Refactor install commands

### LINKED ISSUE #5: [request] Separate commands `install kops` & `install kubectl`
### Orb version

0.3.0

### What happened

Sometimes we just need to install `kubectl` without `kops`, the current command can't do that, because both of them are in 1 command.

### Expected behavior

Separates into 2 commands, so when you only need `kubectl`, it can save execution time.

### PR DESCRIPTION
**Pull request by duffn**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

Refactors the install command to separate kops from kubectl as requested in 
#5.

### Description

Refactors the install command to separate `install-kops` from `install-kubectl` commands.

Closes #5.

### CODE REVIEW

**Comment by nvanheuverzwijn** [comment]
@duffn did you check the file `.circleci/config.yaml` ? There are test out there that probably need to be changed considering this patch.

**Comment by duffn** [comment]
@nvanheuverzwijn Thanks for pointing that out. I've updated the file.

**Comment by duffn** [comment]
@lokst I'm not entirely sure that I did that right (can I just call other commands like this?), but I've pushed a new commit and updated the tests.

File: src/commands/install.yml [hunk 1]
```diff
@@ -13,48 +13,8 @@ parameters:
 steps:
   - run:
```
Comment (lokst) [inline src/commands/install.yml]: The correct way to reference the `install-kops` and `install-kubectl` commands would be like this:

```
steps:
  - install-kops
  - install-kubectl
```

Comment (duffn) [reply src/commands/install.yml]: Thanks @lokst. I've updated.

**Comment by lokst** [comment]
@duffn There was an additional issue with the `install` command that I fixed in https://github.com/CircleCI-Public/kubernetes-orb/pull/13. The changes are ready to be used via `circleci/kubernetes@0.6.0` !

**Comment by duffn** [comment]
Excellent, thank you!

**Pull request merged**